TDDUnitTests/UserAutoFixture.cs

```diff
@@ -0,0 +1,21 @@
+﻿using AutoFixture;
+using TDDProject.Models;
+
+namespace TDDUnitTests
+{
+    public static class UserAutoFixture
+    {
+        public static Fixture GetUserFixture(Fixture _fixture)
```

Comment:
this is bad
why do you even need this 
please read and learn about auto fixture.